pmultiqc/modules/quantms/histogram.py

```
@@ -28,106 +29,108 @@
~                if str(value) in self.bins:
~                    self.data[str(value)][stack] += 1
~                else:
~                    self.bins.append(str(value))
~                    if self.stacks:
~                        self.data[str(value)] = dict.fromkeys(self.stacks, 0)
~                        self.data[str(value)][stack] = 1
                     else:
                         self.data[str(value)] = {stack: 1}
                 # sort
-                if type(value) != str and not self.stacks:
+                if isinstance(value) != str and not self.stacks:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)